esprima.js

```diff
@@ -11,6 +11,11 @@
 var esprima = require('esprima-fb');
 var estraverse = require('estraverse-fb');
 var escodegen = require('escodegen');
+var listver = require('listver');
+
+var webpackMajorVersion = parseInt(
```

Comment:
Also you can find all of the API endpoints for the `loaderContext` at https://webpack.js.org/api/loaders/#the-loader-context